### ISSUE TITLE
What should search box in standard header search?

### ISSUE THREAD

**Issue by timwis** [CityOfPhiladelphia/patterns#4]
Do we generally want people to use our standard header, which includes the search box? If so, what should that search box search? All of phila.gov? (If so, do we need to provide some typeform code?)

What if it's a separate application/site that has its own content/search functionality? We could have them use this search box, but will users expect it to work for all of phila.gov?

Personally, I'm going to use the standard header *without* the search box since my application *does not* support searching, and I worry users will think they can search my application if I give them a search box.

**Comment by gsf** [comment]
If the search is included I would definitely expect it to search all of phila.gov. But I agree it could be confusing on some application pages.

The site-wide search serves to tie the application back to the rest of the site. Without it, do we need more links to other content? Maybe our footer pattern should be expanded to include those?

**Comment by atogle** [comment]
Agree with @gsf. Header search should search the site, but we should have a header pattern that does not include a search field.

I'm talking with @majshehab tomorrow about designs for a "fat footer" as a way to have more links to other content.

**Comment by majshehab** [comment]
The search box acts as "I tried browsing for information I need, but couldn't find it. Let me search the rest of the domain."  Changing the text in the search box to "Search alpha.phila.gov" may help clarify the possible confusion.

I agree that it may be confusing to a user to have it on a page with a non-searchable application, but it also supplements their experience if the application fails them.  Maybe I browsed to a property application looking for (something)... and after I input my address, I didn't find that information.  My next step as a user would be to search the rest of the website - should I have to go to another page to do that?  Since it is a common web practice to have a search box on the top right of the header, maybe we should test this?
